### PR TITLE
A step towards pep8

### PR DESCRIPTION
**Pull request by dunnen**
I fixed the white space issues, newline issues, and comment format issues.

### CODE REVIEW

**Comment by dunnen** [comment]
Does anyone know why I keep getting CI failures? I ran CI before I did the PR on my own branch and it [passed](https://travis-ci.org/dunnen/arky/builds/343216370) but when I create a PR test_check_latency_good fails.

**Comment by roks0n** [comment]
This is odd, locally when I run tests from your branch using py2.7 tests pass. The failed tests means that latency has a value of  `False` or `None`. In the case of the failed tests I assume it's None because an exception was raised here: https://github.com/ArkEcosystem/arky/blob/aip11/arky/rest.py#L138-L141

**Comment by dunnen** [comment]
That's a good point. The connection might be so fast that 0 is being returned.

File: test/test_rest.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_check_latency_good(self):
             json={}
         )
         latency = arky.rest.check_latency(peer)
-        assert latency
+        assert latency is not None
```
Comment (roks0n) [inline test/test_rest.py]: I'm curious what the output is if isn't None or int. Any ideas?

Comment (roks0n) [reply test/test_rest.py]: Ah, it's a float not an int! :)

Comment (dunnen) [reply test/test_rest.py]: I think it's a datetime.timedelta. The following code in in the [requests|http://docs.python-requests.org/en/master/_modules/requests/sessions/] lib code:
```python
# Total elapsed time of the request (approximately)
elapsed = preferred_clock() - start
r.elapsed = timedelta(seconds=elapsed)
```

Comment (dunnen) [reply test/test_rest.py]: Ah yes. You're right. elapsed is a datetime.timedelta object but the total_seconds() returns a float.

Comment (dunnen) [reply test/test_rest.py]: Should I leave it only checking "is not None" or should I add a float check back in?

Comment (roks0n) [reply test/test_rest.py]: It's not critical, but personally, I think I would add it. :)

**Comment by Moustikitos** [comment]
Welcome dunnen, thanks for your contribution !

**Pull request merged**